pretext/ExtraTopics/Glossary.ptx

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-glossary">
+  <title>Glossary</title>
+
+  
+  <!-- BOILERPLATE STUFF -->
+  <glossary sorted="False">
+    <gi>
+      <title>activecode</title>
+      <p>A unique interpreter environment that allows Python to be executed from within a web browser.</p>
+    </gi>
+    <gi>
+      <title>algorithm</title>
+      <p>A general step by step process for solving a problem.</p>
+    </gi>
+    <gi>
+      <title>bug</title>
+      <p>An error in a program.</p>
+    </gi>
+    <gi>
+      <title>byte code</title>
+      <p>An intermediate language between source code and object code. Many
+                            modern languages first compile source code into byte code and then
+                            interpret the byte code with a program called a <em>virtual machine</em>.</p>
+    </gi>
+    <gi>
+      <title>codelens</title>
+      <p>An interactive environment that allows the user to control the step by step execution of a Python program</p>
+    </gi>
+    <gi>
+      <title>comment</title>
+      <p>Information in a program that is meant for other programmers (or anyone
+                            reading the source code) and has no effect on the execution of the
+                            program.</p>
+    </gi>
+    <gi>
+      <title>compile</title>
+      <p>To translate a program written in a high-level language into a
+                            low-level language all at once, in preparation for later execution.</p>
+    </gi>
+    <gi>
+      <title>debugging</title>
+      <p>The process of finding and removing any of the three kinds of
+                            programming errors.</p>
+    </gi>
+    <gi>
+      <title>exception</title>
+      <p>Another name for a runtime error.</p>
+    </gi>
+    <gi>
+      <title>executable</title>
+      <p>Another name for object code that is ready to be executed.</p>
```

Comment:
This is not correct. 

Executable should be defined as code that is ready to be run on the target device.